core/src/main/res/layout/dev_menu_fragment_logs.xml

```diff
@@ -55,14 +58,24 @@
             android:requiresFadingEdge="horizontal"
             android:fadingEdgeLength="70dp">
 
-            <TextView
+            <TextSwitcher
```

Comment:
Interesting, I haven't seen this component before.  Is the purpose here just to add the fade animation when the text changes?

Reply:
Yes, the TextSwitcher switches between displaying the text on its two child TextViews and animates the action of displaying new text.